plugins/modules/ini_file.py

```diff
@@ -179,16 +185,39 @@
       - pepsi
     mode: '0600'
     state: present
+
+- name: remove the peer configuration for 10.128.0.11/32
+  community.general.ini_file:
+    path: /etc/wireguard/wg0.conf
+    section: Peer
+    section_has:
+      - option: AllowedIps
+      - value: 10.4.0.11/32
```

Comment:
Shouldn't this be:
```suggestion
      - option: AllowedIps
        value: 10.4.0.11/32
```
? Same below.